File: test/fixtures/python/bash_redirect_herestring.py
import requests

headers = {
    '\'Accept\'': '\'application/json\'',
    'Authorization': 'Bearer 000000000000000-0000',
    'Content-Type': 'application/json',
}

data = '$MYVARIABLE'

response = requests.post('http://localhost:28139/api/servers/00000000000/shared_servers/', headers=headers, data=data)

File: test/fixtures/r/bash_redirect_herestring.r
library(httr)

headers = c(
  `'Accept'` = "'application/json'",
  Authorization = "Bearer 000000000000000-0000",
  `Content-Type` = "application/json"
)

data = upload_file("-")
res <- httr::POST(url = "http://localhost:28139/api/servers/00000000000/shared_servers/", httr::add_headers(.headers=headers), body = data)
